tests/lib/cli-engine/cli-engine.js

```diff
@@ -3032,6 +3031,320 @@ describe("CLIEngine", () => {
             });
         });
 
+        describe("multiple processors", () => {
+
+            /**
+             * Unindent template strings.
+             * @param {string[]} strings Strings.
+             * @param  {...any} values Values.
+             * @returns {string} Unindented string.
+             */
+            function unindent(strings, ...values) {
```

Comment:
Surely we have this code available somewhere else (e.g., `indent` rule tests). Could we make a test utility for this?